registry/app.go

```diff
@@ -88,6 +88,8 @@ func NewApp(configuration configuration.Configuration) *App {
 }
 
 func (app *App) ServeHTTP(w http.ResponseWriter, r *http.Request) {
+	// Set a header with the Docker Distribution API Version for all responses.
+	w.Header().Set("Docker-Distribution-API-Version", "2.0")
```

Comment:
Thanks @thaJeztah 
Disclosing the exact version of a service is usually considered a security issue.
We will certainly add specific endpoints that will provide extended informations such as that, but they will have to be explicitly enabled.
This, on the other hand, is about being able to more simply distinguish major API versions for the clients. 
Thanks again for your input on this.